docs/reference/files.md

```diff
@@ -716,3 +1026,36 @@ mutation {
 	}
 }
 ```
+
+</template>
+<template #sdk>
+
+```js
+import { createDirectus } from '@directus/sdk';
+import { staticToken } from '@directus/sdk/auth'; // OMIT
```

Comment:
What does the `OMIT` comment mean?

Reply:
This is meant to be nipped out when adding to docs. Bev’s been using a static token auth for testing, but I want us to omit auth code in examples.